instructors/instructor-notes.md

```diff
@@ -19,47 +19,53 @@ Details about the Instructor Trainer role including the application process, dut
-- If teaching in person:
-  - Confirm with your host that breakout rooms are available or make an alternate plan
-  - Create a plan for printing handouts \& determine who will bring sticky notes
+- For online events:
+  - Test videoconferencing set up using login credentials provided.
+  - Make a copy of the [Virtual Minute Cards template][minute-cards-template] and personalize for your event.
+  - Plan how you will have trainees watch videos. If you plan to play them on zoom, familiarize yourself with audio settings and consider an ad blocking strategy.
+- For in-person events:
+  - Confirm that breakout rooms are available or make an alternate plan
+  - Determine who will bring paper (if needed for concept mapping) and sticky notes (for minute cards & demonstration).
   - Make sure you are prepared for the audiovisual setup in your room with the correct dongles/connectors etc.
-  - Check on availability/timing of breaks, lunch, or any other details that matter to you
+  - Check on availability/timing of breaks, lunch, etc.
+  - Familiarize yourself with the room, building, and accessibility resources as possible.
 
 ### During the event
 
-- Take attendance.
-- Remind member trainees to fill out application (Open Training applicants have already done this).
-- Remind trainees to sign up for demo, discussion (links in [checkout checklist][checkout-checklist]).
+- Take attendance. When marking trainees absent due to missed time, be sure to add notes describing what time was missed.
 - Monitor the Etherpad / Google Doc for questions and responses to exercises.
-- If teaching remotely: Ensure that you are logged into the Zoom room with the host key so that you can mute participants and create breakout rooms as needed.
-- If teaching remotely: Turn off video during long exercises and breaks and disconnect during lunch (note: you may need to re-log in with the host key if returning after disconnecting).
+- For online events, ensure that you are logged into the Zoom room with the host key so that you can mute participants and create breakout rooms as needed.
+- For self-organised training events, it may be necessary to remind trainees to fill out the application.
+
 
 ### Immediately after the event
 
-- Send a list of those who completed the training to checkout@carpentries.
-- Send an email to trainees thanking them for participating and linking to [checkout checklist][checkout-checklist]. See [template][post-template].
+- Double check that your attendance sheet is complete. For self-organized events, email instructor.training@carpentries.org to be sure we have your attendance record.
+- If you wish, you can an email to trainees thanking them for participating. 
```

Comment:
@ndporter the checkout checklist that was linked here is very outdated. At this time we feel that our pre-training email and follow-up reminder (sent ~2 months after training) are sufficient in providing all the necessary information that trainees will need.